pkg/client/typed/dynamic/client.go

```diff
@@ -65,10 +65,10 @@ func NewClient(conf *restclient.Config) (*Client, error) {
 	}
 
 	if conf.QPS == 0.0 {
-		conf.QPS = 5.0
+		conf.QPS = restclient.DefaultQPS
 	}
 	if conf.Burst == 0 {
-		conf.Burst = 10
+		conf.Burst = restclient.DefaultBurst
```

Comment:
Do we really need this? Could we add this to RESTClientFor (if it doesn't already) and avoid putting this logic in every single generated client?